code/modules/food_and_drinks/recipes/recipes_microwave.dm

```diff
@@ -140,6 +140,61 @@
 	)
 	result = /obj/item/reagent_containers/food/snacks/cheeseburger
 
+/datum/recipe/microwave/appendixburger
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/organ/internal/appendix
+	)
+	result = /obj/item/reagent_containers/food/snacks/appendixburger
+
+/datum/recipe/microwave/appendixburger_bitten
+	items = list(
+		/obj/item/reagent_containers/food/snacks/appendix,
+		/obj/item/reagent_containers/food/snacks/bun
+		)
```

Comment:
Adujst the identation for all the recipes items please (look other recipes in this file and follow their style).

Reply:
should be cleared up now